### ISSUE TITLE
[Feature Request] Personalised Pre-Defined Values

### ISSUE THREAD

**Issue by ordric** [olivierkes/manuskript#722]
Hi, I have recently installed this software, and so far it is looking very good, but I have a feature request...

In the 'Detailed Info' for characters, you can add any description for any value you wish to type in. The problem is that you have to do this for every character. Would it be possible for us to have the ability to declare in a project, common data items, so that when you go to the 'Detailed Info', the values are already there for you to fill in.

I would see this as some form of configuration textfile which would get loaded each time the project was opened, with one tag name per line, such as:

Birthday
Birthplace
Mother
Father
Eyecolour

It would be even better if you could declare specific values and have a drop-down list, to select from.
For example, say I want to record for all charachers a value called 'status', and in this value there are four options: "Patrician", "Equestrian", "Plebean" and "Slave".

Would that be possible?

**Comment by gedakc** [comment]
This request looks like a duplicate of issue #252.

**Comment by ordric** [comment]
No, I don't believe it is. What I was asking for, was just the ability to be able to declare custom global fields for characters on a project by project basis. Not a Character Template.

**Comment by AlanDunaway** [comment]
> No, I don't believe it is. What I was asking for, was just the ability to be able to declare custom global fields for characters on a project by project basis. Not a Character Template.

I'd second that along with organization. Pre-adding lines in Detailed Info would be awesome for fiction.
